tasks/image-segmentation/task.yaml

```diff
@@ -11,6 +11,11 @@ param_groups:
     value: null
     data_type: dataset
     required: true
+  - name: trainedModel
+    description: Model id used for image segmentation.
+    value: null
+    data_type: model
+    required: true
```

Comment:
Can you go through the parameters and see which ones are only required for either training or validation and set `required` to false for them. Otherwise the user won't be allowed to start a validation run without filling out all the training related parameters.